workshops/ec2-auto-scaling-with-multiple-instance-types-and-purchase-options/spot-interruption-handler.yaml

```diff
@@ -0,0 +1,119 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Metadata:
+  License:
+    Description: 'Copyright 2019 Amazon.com, Inc. and its affiliates. All Rights Reserved.
```

Comment:
You should definitely add yourself as an author on this one (waiting for you to kill me in 3,2,1,...)